src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/CustomDatasetSource.kt

```diff
@@ -59,14 +60,25 @@ class CustomDatasetSource private constructor(
             .build()
     }
 
+    @Deprecated(message = "Use CustomDatasetSource.storeInS3(location, jira) instead",
+        replaceWith = ReplaceWith("store(location, null)")
```

Comment:
If `store(location, null)` is `internal` would that work?

Reply:
Indeed it wouldn't